Spreadsheet/style.css

```diff
@@ -74,29 +73,26 @@ header {
     overflow: scroll;
 }
 
-.box {
+.cell {
     border: 1px solid #01A9B4;
     width: 100px;
     height: 30px;
     text-align: center;
     line-height: 30px;
 }
 
-.box-border {
+.selectedCell {
     border: 1px solid orange;
     background-color: white;
     font-size: 15px;
     color: black;
     text-align: center;
 }
 
-.parentBoxColor {
+.selectedHeader {
```

Comment:
same idea here, `parent` is a bit abstract and doesn't really indicate any specific concept, just that it's the parent of something in the page (which could be anything). In reality this is the style applied to column and row header cells when they are selected so `selectedHeader` here is more descriptive.